inc/tasks/myalerts.php

```diff
@@ -21,7 +20,7 @@ function task_myalerts($task)
 		$lang->load('myalerts');
 	}
 
-	if ($db->delete_query('alerts', 'unread = 0')) {
+	if ($db->delete_query('alerts', 'unread = 0 AND dateline <= DATESUB(NOW(), INTERVAL 120 DAY)')) {
```

Comment:
Instead of using SQL commands to get the date, you need to you PHP and `TIME_NOW` which is more efficient, and also adjusts for server time stamp.